tests/qa/cw/config_example.yaml

```diff
@@ -0,0 +1,13 @@
+api: "http://localhost:1317"
+grpc: "localhost:9090"
+rpc: "http://localhost:26657"
+chain_id: "test-1"
+mnemonics:
+  - "copper push brief egg scan entry inform record adjust fossil boss egg comic alien upon aspect dry avoid interest fury window hint race symptom"
+  - "maximum display century economy unlock van census kite error heart snow filter midnight usage egg venture cash kick motor survey drastic edge muffin visual"
+  - "banner spread envelope side kite person disagree path silver will brother under couch edit food venture squirrel civil budget number acquire point work mass"
+  - "veteran try aware erosion drink dance decade comic dawn museum release episode original list ability owner size tuition surface ceiling depth seminar capable only"
+  - "vacuum burst ordinary enact leaf rabbit gather lend left chase park action dish danger green jeans lucky dish mesh language collect acquire waste load"
+  - "open attitude harsh casino rent attitude midnight debris describe spare cancel crisp olive ride elite gallery leaf buffalo sheriff filter rotate path begin soldier"
+  - "alley afraid soup fall idea toss can goose become valve initial strong forward bright dish figure check leopard decide warfare hub unusual join cart"
+  - "record gift you once hip style during joke field prize dust unique length more pencil transfer quit train device arrive energy sort steak upset"
```

Comment:
how about using a map of account name -> mnemonic, and then use account names, rather than indexes.